app/models/course/assessment/answer/multiple_response.rb

```diff
@@ -5,4 +5,25 @@ class Course::Assessment::Answer::MultipleResponse < ActiveRecord::Base
   has_many :answer_options, class_name: Course::Assessment::Answer::MultipleResponseOption.name,
                             dependent: :destroy, foreign_key: :answer_id, inverse_of: :answer
   has_many :options, through: :answer_options
+
+  # Multiple Response Questions (MRQs) options are displayed in forms using radio buttons if they
+  # are Multiple Choice Questions (MCQs), otherwise, they are displayed using checkboxes. This
+  # method allows the radio button for the selected MCQ option to be marked as checked when the
+  # form is generated for display.
+  #
+  # In both the MCQ and non-MCQ cases, the options are rendered using the same custom simple_form
+  # input component which delegates the rendering to either a radio button or checkbox renderer
+  # as appropriate. The radio button renderer expects the attribute (i.e. #option_ids) to return
+  # an element, not an array of elements.
+  def override_option_ids_method_for_form_display!
+    def self.option_ids
```

Comment:
@kxmbrian  I am going to remove this because it's going to be global once you called it.